microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -153,125 +154,141 @@ Visual Studio 2019 version 16.9 or later is required, for this tutorial.  Visual
 Next, you set up this new WinUI 2 (UWP) project to host the WebView2 control and use the WebView2 API.
 
 
-<!-- maintenance link; keep: main copy:
-[Install the WebView2 SDK](../how-to/machine-setup.md#install-the-webview2-sdk) in _Set up your Dev environment for WebView2_
--->
 <!-- ====================================================================== -->
-## Step 5 - Install the prerelease WebView2 SDK
+## Step 5 - Install the WinUI 2 SDK (Microsoft.UI.Xaml)
+
+Next, you install the **Microsoft.UI.Xaml** package for this project.  Microsoft.UI.Xaml is WinUI 2.
 
 1. In Solution Explorer, right-click the project (not the solution node above it), and then select **Manage NuGet Packages**.
 
    The **NuGet Package Manager** panel opens in Visual Studio.
 
 1. In the **NuGet Package Manager**, click the **Browse** tab.
 
-1. To the right of the search text box, select the **Include prerelease** check box.
+1. Clear the **Include prerelease** check box.
 
-1. In the search text box, enter **Microsoft.Web.WebView2**.
+1. In the **Search** box, enter **Microsoft.UI.Xaml**, and then select the **Microsoft.UI.Xaml** card below the search box:
 
-   The **Microsoft.Web.WebView2** card appears in the search results.
+   ![The NuGet package manager to install Microsoft.UI.Xaml](media/winui2-nuget-package-ui-xaml.png)
 
-1. Click the **Microsoft.Web.WebView2** card below the search box.
+1. On the right, make sure that the **Version** is **Latest stable**.
 
-1. On the right, in the **Version** dropdown list, make sure **Latest prerelease** is selected:
+1. Click the  **Install** (or **Update**) button.
 
-   ![NuGet Package Manager with WebView2 SDK prerelease selected.](media/winui2-nuget-pkg-mgr-webview2.png)
+   The **Preview Changes** dialog box appears:
 
-   _To zoom, right-click > **Open image in new tab**._
+   ![The 'Preview Changes' dialog box for installing the Microsoft.UI.Xaml package.](media/winui2-preview-changes-ui-xaml-pkg.png)
 
-1. Click the **Install** (or **Update**) button.
+   Note: The capture shows 2.7.1, before General Availability of WebView2 in the Microsoft.UI.Xaml NuGet package.
```

Comment:
@jasonstephen15 - should this text state "2.8 or greater"?  what's the version # of G.A. of the NuGet pkg for WinUI 2?

Reply:
Updated captures in other article in this PR, shows version numbers used for Prerelease of WinUI 2 SDK:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/samples/webview2_sample_uwp?branch=pr-en-us-2043
Clarified the Get Start article which shows 2.7 captures, will have to update those after GA; stated 2.8+ required.
So, Resolving conversation.